signbank/dictionary/admin.py

```diff
@@ -183,6 +183,8 @@ class GlossAdmin(VersionAdmin):
     fieldsets = ((None, {'fields': ('dataset', 'published', 'exclude_from_ecv', 'id', 'idgloss', 'idgloss_mi', 'notes', 'hint', 'signer', 'filmbatch')},),
                  (_('Created/Updated'), {'fields': ('created_at',
                   'created_by', 'updated_at', 'updated_by')},),
+                 (_('Morphology'), {'fields': ('inflection1', 'inflection2', 'inflection3'),
```

Comment:
These column names are not descriptive, and it sounds like they can be, since they have descriptive labels in the model definition.

I would suggest:

* `inflection_temporal`
* `inflection_manner_degree`
* `inflection_plural`

Reply:
Sure thing.